lib_comfyui/parallel_utils.py

```diff
@@ -50,17 +50,23 @@ def __setstate__(self, state):
         *state, self._consumer_ready_event, self._producer = state
         return super(SynchronizingQueue, self).__setstate__(state)
 
-    def start_thread(self):
+
+class AsyncProducerHandler:
```

Comment:
`AsyncProducerHandler` gives me the impression this is implemented using an asyncio loop, where it is in fact implemented using a thread. I think it should have `Thread` in the name somewhere instead.

Reply:
Although it isn't a thread, but it isn't a handler either

Reply:
`ProducerThreadHandler`?

Reply:
Maybe just `ProducerHandler` actually. I think I see how this could be considered a "handler", although IIUC a handler usually refers to just the callback part, not the setting up a thread and stopping it part.

Reply:
I honestly don't know how to call this better haha